README.md

```diff
@@ -118,3 +118,27 @@ app.post("/route", async (req, res) => {
 
 ```
 Note: During routine usage, there are no special calls necessary to make use of Layer8. The res.json() & res.send() & res.end() have been overwritten by Layer8 and will be used automatically. 
+
+
+## Setup Metrics Collector
+
+### Prerequisite
+- Docker up and running
+- Docker compose
+
+### Setup
+
+1. Start InfluxDB v2 as a Docker container by running the following command:
+```
+docker compose -f docker-compose-influx.yml up 
+```
+2. Open the InfluxDB dashboard via a browser using the defined credentials in docker-compose-influx.yml on port 8086.
+3. Create an access token in the InfluxDB UI (https://docs.influxdata.com/influxdb/cloud/admin/tokens/create-token/).
+4. Create a bucket with `layer8` name in InfluxDB
+5. Add `INFLUXDB_URL` and `INFLUXDB_TOKEN` variable to `.env` file based on the created token.
```

Comment:
Please update the .env.dev file accordingly, so it is easier for the team members to know which environment variables should be made.